samples/apps/copilot-chat-app/scripts/1_install-reqs.cmd

```diff
@@ -0,0 +1,5 @@
+:: Install chocolatey
+@"%SystemRoot%\System32\WindowsPowerShell\v1.0\powershell.exe" -NoProfile -InputFormat None -ExecutionPolicy Bypass -Command "[System.Net.ServicePointManager]::SecurityProtocol = 3072; iex ((New-Object System.Net.WebClient).DownloadString('https://community.chocolatey.org/install.ps1'))" && SET "PATH=%PATH%;%ALLUSERSPROFILE%\chocolatey\bin"
+
+:: Install prereqs
+choco install -y dotnet-6.0-sdk nodejs yarn
```

Comment:
This looks like the only Windows-specific line in your scripts. If we could provide cross-platform support here and use PowerShell 7 (pwsh), we could get cross-platform support for free.